packages/wrangler/vitest.config.mts

```diff
@@ -61,7 +61,7 @@ export default defineConfig({
 		testTimeout: 15_000,
 		pool: "forks",
 		retry: 0,
-		include: ["**/__tests__/**/*.test.ts"],
+		include: ["**/__tests__/**/*.test.ts", "**/__tests__/**/*.test.tsx"],
```

Comment:
not sure if we want to be as lax as `**/__tests__/**/*.test.ts*` here 🤔 ... so went for the more explicit & restrictive option. Happy to change if there are other strong opinions abt it

Reply:
This is clearer 👍